### PR TITLE
Added documentation about n:ifcontent

### PR DESCRIPTION
**Pull request by sallyx**

### CODE REVIEW

**Comment by dg** [comment]
Thanks

File: en/default-macros.texy [hunk 1]
```diff
@@ -52,6 +52,8 @@ Are you interested in taking a closer look at [Latte syntax |templating#latte]?
 |---------------------------
 | `n:class`                      |  [smart class attribute |#n-class]
 | `n:attr`                       |  [smart HTML attributes |#n-attr]
+| `n:ifcontent`                  |  [Ommit empty HTML tag |#n-ifcontent]
+| `n:tag-if`                     |  Ommit HTML tag if condition is FALSE
```
Comment (Majkl578) [inline en/default-macros.texy]: Typo, ``omit`.

File: en/default-macros.texy [hunk 2]
```diff
@@ -401,6 +403,29 @@ Depending on the values of the variables `$value` and `$checked` this will print
 <input type="checkbox" value="Hello" checked>
 \--
 
+n:ifcontent
+-----------
+
+`n:ifcontent` macro remove empty tag
```
Comment (Majkl578) [inline en/default-macros.texy]: removes

Comment (dg) [inline en/default-macros.texy]: This is merged, send pr please.

**Pull request merged**